src/modules/feature-modules/accessor/pages/innovation/support/organisations-support-status-suggest.component.ts

```diff
@@ -20,109 +33,420 @@ import { SupportLogType } from '@modules/shared/services/innovations.dtos';
 export class InnovationSupportOrganisationsSupportStatusSuggestComponent extends CoreComponent implements OnInit {
   private supportUpdateSideEffect = false;
 
-  currentStep: 1 | 2 = 1;
+  stepNumber: number;
   innovation: ContextInnovationType;
 
-  form = new FormGroup(
+  form: FormGroup<{
+    organisation: FormControl<null | string>;
+    units?: FormArray<FormControl<string>>;
+    comment: FormControl<null | string>;
+  }> = new FormGroup(
     {
-      organisationUnits: new FormArray<FormControl<string>>([]),
-      comment: new FormControl<string>('', CustomValidators.required('A comment is required')),
-      confirm: new FormControl<boolean>(false, CustomValidators.required('You need to confirm to proceed'))
+      organisation: new FormControl<null | string>(null, [
+        CustomValidators.required('Select an organisation and click continue')
+      ]),
+      comment: new FormControl<string>(
+        '',
+        CustomValidators.required(
+          'Describe why you think this innovation would benefit from support from this organisation and click continue'
+        )
+      )
     },
     { updateOn: 'blur' }
   );
 
-  groupedItems: Required<FormEngineParameterModel>['groupedItems'] = [];
+  organisations: OrganisationsListDTO[] = [];
+  organisationsToSuggest: (OrganisationsListDTO & { description: string | undefined })[] = [];
+
+  organisationItems: Required<FormEngineParameterModel>['items'] = [];
+  unitsItems: Required<FormEngineParameterModel>['items'] = [];
 
   chosenUnits: {
-    list: { organisation: string; units: string[] }[];
+    organisation: { name: string; acronym: string; information?: OrganisationInformation };
+    unitsNames: string[];
     values: string[];
-  } = { list: [], values: [] };
+  } = { organisation: { name: '', acronym: '' }, unitsNames: [], values: [] };
 
-  submitButton = { isActive: true, label: 'Confirm and notify organisations' };
+  submitButton = { isActive: true, label: 'Confirm' };
 
-  isQualifyingAccessorRole = false;
+  organisationsInformation: OrganisationInformation[] = [
```

Comment:
This could be a `Map` where the key is `acronym` instead of an array, would help on the `finds` for this information.
 
We could extract this information to a file `data/organisations-information.data.ts` inside the current directory if you prefer.

Reply:
We are going to update the referral criteria page with the same content at some point in the future (new user story), so I didn't wanted to change things outside the scope of this user story. But yes, it's better to move it to another file in the future so both components have access to it, I'll leave a comment

Reply:
Fine by me.